### PR TITLE
BiomeArea is now BiomeVolume and all that this implies

### PR DESCRIPTION
**Pull request by DDoS**
Needs a thorough review.

See: https://github.com/SpongePowered/SpongeCommon/pull/967
See: https://github.com/SpongePowered/SpongeForge/pull/938

### CODE REVIEW

File: src/main/java/org/spongepowered/api/world/extent/BiomeVolume.java [hunk 1]
```diff
@@ -25,165 +25,164 @@
...
     }
 
     /**
      * Gets the {@link BiomeType} at the given location.
      *
      * @param x The X position
-     * @param z The Z position
-     * @return The biome
+     * @param y The Y position
+     * @param z The Z position  @return The biome
```
Comment (Deamon5550) [inline src/main/java/org/spongepowered/api/world/extent/BiomeVolume.java]: need a newline

**Comment by DDoS** [comment]
This can be added to the small issue list.

File: src/main/java/org/spongepowered/api/world/extent/BiomeVolume.java (hunk 1 above)
Comment (DDoS) [inline src/main/java/org/spongepowered/api/world/extent/BiomeVolume.java]: This can be added to the small issue list.

**Comment by kashike** [comment]
https://github.com/SpongePowered/SpongeAPI/commit/9e7975e3f7173b4600bd1a10795743827e70a86d

**Pull request closed without merging**